osbs/cli/main.py

```diff
@@ -422,14 +417,8 @@ def cli():
 
 def main():
     parser, args = cli()
-    try:
-        os_conf = Configuration(conf_file=args.config,
-                                cli_args=args)
-    except OsbsException as ex:
-        logger.error("Configuration error: %s", ex.message)
-        return -1
 
-    is_verbose = os_conf.get_verbosity()
+    is_verbose = args.verbose
```

Comment:
Do we only want to get verbosity from the CLI, not from the config file?

(the TBD comment about that was in the prev. commit, but I don't know if we were sure about it or if it was just an idea)